<?xml version="1.0"?>
<launch>
    <node name="sick_lrs_4xxx" pkg="sick_scan" type="sick_generic_caller" respawn="false" output="screen">
        <param name="scanner_type" type="string" value="sick_lrs_4xxx"/>
        <param name="hostname" type="string" value="192.168.0.1"/>
        <param name="cloud_topic" type="string" value="cloud"/>
        <param name="frame_id" type="str" value="cloud"/>
        <param name="port" type="string" value="2112"/>
        <param name="timelimit" type="int" value="5"/>
        <param name="min_ang" type="double" value="-4.71238898"/> <!-- value from sick_scan2/config/sick_lrs_4xxx.yaml -->
        <param name="max_ang" type="double" value="1.57079632"/>  <!-- value from sick_scan2/config/sick_lrs_4xxx.yaml -->
        <param name="use_binary_protocol" type="bool" value="true"/>
        <param name="range_max" type="double" value="100.0"/>
        <param name="intensity" type="bool" value="True"/>
        <param name="min_intensity" type="double" value="0.0"/> <!-- Set range of LaserScan messages to infinity, if intensity < min_intensity (default: 0) -->
    </node>
</launch>
